spec/features/edit_sources_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature 'Edit Source' do
+  scenario 'Importer edits source' do
+    better_source = Fabricate(
+      :source,
+      email_rank: 1,
+      location_rank: 1,
+      organization_name_rank: 1,
+      phone_number_rank: 1,
+      website_rank: 1
+    )
+
+    source = Fabricate(
+      :source,
+      email_rank: 2,
+      location_rank: 2,
+      organization_name_rank: 2,
+      phone_number_rank: 2,
+      website_rank: 2
+    )
+
+    visit "/sources/#{source.id}/edit"
+
+    fill_in 'Name', with: 'New Name'
+
+    first_option = "1 - insert above #{better_source.name}"
```

Comment:
I was confused by this. So ... better source becomes less better source? If I'm reading this correctly, I wonder if we should switch the declarations of `better_source` and `source`. So `better_source` would start out at level `2`, which the reader of this spec would immediately recognize as incorrect vis-a-vis the next source. Then we can imagine our imaginary web user going to this page and saying, "wait, Foo is actually the better source, I should correct this." And at the end of the fable, `better_source` wins and is :1st_place_medal:.